packages/transaction-controller/src/types.ts

```diff
@@ -354,3 +360,8 @@ export interface DappSuggestedGasFees {
   maxFeePerGas?: string;
   maxPriorityFeePerGas?: string;
 }
+
+export interface SendFlowHistoryEntry {
```

Comment:
Shall we use a `type` if only for consistency with the other types in this file?

And although it will likely overlap with the great property description, should we add a description here also?

Reply:
Sorted in https://github.com/MetaMask/core/pull/1665/commits/ccb36fd46c67776efda5b2ae5f96df02a79e5fad